pkgs/tools/package-management/nix/README.md

```diff
@@ -0,0 +1,34 @@
+# Steps for Testing New Releases
+
+## Patch Releases or Updating `nixVersions.{latest,git}`
+
+Branch to merge into: **master** or **release-$nixos-version**
+
+Build the affected Nix packages and run their tests on the following platforms: **x86_64-linux**, **aarch64-linux**, **x86_64-darwin**, and **aarch64-darwin**.
+If you lack the necessary hardware for these platforms, you may need to ask others for assistance with the builds.
+Alternatively, you can request access to the Nix community builder for all platforms [here](https://github.com/NixOS/aarch64-build-box) and [here](https://nix-community.org/community-builder/).
+
+To build all dependent packages, use:
+
+```
+nix-review pr <your-pull-request>
+```
+
+And to build all important NixOS tests, run:
+
+```
+# Replace $version with the actual Nix version
+nix-build nixVersions.nix_$version.tests
+```
+
+Be sure to also update the `nix-fallback-paths` whenever you do a patch release for `nixVersions.stable`
+
+```
+# Replace $version with the actual Nix version
+curl https://releases.nixos.org/nix/nix-$version/fallback-paths.nix > nixos/modules/installer/tools/nix-fallback-paths.nix
+```
+
+## Major Version Bumps
+
+If you're updating `nixVersions.stable`, follow all the steps mentioned above, but use the **staging-next** branch for your pull request.
```

Comment:
Probably best to target `staging` in general.

In the case of the current 2.24 bump it was done during this bump the staging-next cycle was at the beginning

but in general someone could attempt to merge in into a staging-next cycle that is about to get merged or during no staging-next cycle.